twitter-sentiment/src/app.py

```diff
@@ -73,10 +94,10 @@ def search_tweets(q: Q):
 async def serve(q: Q):
     if not q.client.initialized:
         init(q)
+        q.client.word = True
         q.client.initialized = True
 
-    if q.args.configure:
-        configure_page(q)
+        configure_page(q, False) if q.args.configure else configure_page(q, True)
```

Comment:
The configure page function is being called whether or not the user has logged in. When I run this locally (where I don't have the credentials setup) this is causing the app to crash. I think you only want to call configure page and search tweets if there are twitter credentials available. The `configure page` function does `dialog = None` so I am never being asked to add my creds locally.